sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2425,12 +2426,10 @@ private MultiLanguageAnalysisInput ConvertToMultiLanguageInputs(IEnumerable<Text
         private LanguageInput ConvertToLanguageInput(string document, string countryHint, int id = 0)
             => new LanguageInput($"{id}", document) { CountryHint = countryHint ?? _options.DefaultCountryHint };
 
-        private static IDictionary<string, string> CreateAdditionalInformation(TextAnalyticsError error)
-        {
-            if (string.IsNullOrEmpty(error.Target))
-                return null;
-            return new Dictionary<string, string> { { "Target", error.Target } };
-        }
+        private static IDictionary<string, string> CreateAdditionalInformation(Error error) =>
```

Comment:
I can't think, help! what is the difference between this and what it had before

Reply:
The incoming type changed from TextAnalytics to Error.    😄

Reply:
The type change here is being reverted due to restoring the inner error extraction that I accidentally removed.